content/docs/user-guide/gems/reference/ros2/_index.md

```diff
@@ -0,0 +1,119 @@
+```
+source /opt/ros/<distro>/setup.bash
+```
+Replace \<distro\> with the ROS 2 distribution name (galactic, humble, ..). Note that you need to re-log for the change to take effect.
+
+#### Additional ROS 2 packages required
+
+* gazebo_msgs: `sudo apt install ros-${ROS_DISTRO}-gazebo-msgs`
+* Ackermann messages: `sudo apt install ros-${ROS_DISTRO}-ackermann-msgs`
+* Control toolbox `sudo apt install ros-${ROS_DISTRO}-control-toolbox`
+* XACRO `sudo apt install ros-${ROS_DISTRO}-xacro`
+
+If a `desktop` installation of ROS 2 distro was selected, everything else should be there.
+
+Use this helpful command to install:
+
+```
+sudo apt install ros-${ROS_DISTRO}-ackermann-msgs ros-${ROS_DISTRO}-control-toolbox ros-${ROS_DISTRO}-nav-msgs ros-${ROS_DISTRO}-gazebo-msgs
+```
+
+## Features
+
+* Direct and natural support of ROS 2 ecosystem:
+  * No bridges. Your simulation node will function as any other ROS 2 node.
+    * This is also good for performance
+  * Easy way to include ROS 2 dependencies.
+* Sensors:
+  * Sensor Component serves as a handy abstraction.
+  * Example implementations of Lidar, Camera, IMU sensors.
+    * Including a few Assets and prefabs which are ready to use. 
+* Automated handling of:
+  * Simulation time, publishing `/clock` supporting non-real time.
+  * Publishing of transformation frames (`/tf`, `/tf_static`).
+  * Validation for topic and namespace names.
+* Robot Control Component:
+  * A quick to use method of controlling your robot with Twist messages.
+  * Can be used with custom LUA scripting. 
+* Vehicle dynamics:
+  * Ackermann Steering, subscribes to message of type [AckermannDrive](http://docs.ros.org/en/api/ackermann_msgs/html/msg/AckermannDrive.html).
+  * Differential drive, subscribes to message of type [Twist](http://docs.ros.org/en/noetic/api/geometry_msgs/html/msg/Twist.html).
+* URDF/XACRO (experimental).
+  
+
+For a "feel" of these features, see an [example project](https://github.com/o3de/RobotVacuumSample) which uses this Gem to run navigation stack.
+
+## Clone this repository
+
+Clone the gem repository from Github:
+
+```
+git clone https://github.com/o3de/o3de-extras
+```
+
+## Building
+
+The Gem is built through building a project which uses it. Make sure to [source your ROS 2 workspace](https://docs.ros.org/en/humble/Tutorials/Configuring-ROS2-Environment.html#source-the-setup-files) before building.
+
+## Adding Gem to your project
+
+To use O3DE ROS 2 Gem in your project, you need to register the Gem with O3DE. First, clone this repository. Then, in O3DE folder:
+```
+scripts/o3de.sh register --gem-path <PATH_TO_CLONED_O3DE_EXTRAS>/Gems/ROS2
+scripts/o3de.sh enable-gem -gn ROS2 -pp <PATH_TO_YOUR_PROJECT>
+```
+
+## Example project
+
+You can test O3DE ROS 2 Gem with [this project](https://github.com/o3de/RobotVacuumSample). It will allow you to run robot navigation. All necessary assets are included.
```

Comment:
Because the project has a clear name, we can reference it by that name. It gives users more context as to where the hyperlink will lead them to.
```suggestion
You can test O3DE ROS 2 Gem with the [Robot Vacuum Sample](https://github.com/o3de/RobotVacuumSample) project. This project allows you to run robot navigation. All necessary assets are included.
```